packages/chakra-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -21,6 +21,7 @@ export default function BaseInputTemplate<
     type,
     value,
     label,
+    displayLabel = true,
```

Comment:
To prevent a breaking change I made `displayLabel` on `WidgetProps` optional, so I default it to true everywhere